globus_cli/commands/endpoint/is_activated.py

```diff
@@ -0,0 +1,68 @@
+import click
+
+from globus_cli.parsing import common_options, endpoint_id_arg
+from globus_cli.safeio import safeprint
+from globus_cli.helpers import outformat_is_json, print_json_response
+from globus_cli.services.transfer import get_client
+
+
+def _print_out(text_message, response_obj):
+    if outformat_is_json():
+        print_json_response(response_obj)
+    else:
+        safeprint(text_message)
+
+
+@click.command('is-activated', short_help='Check if an Endpoint is activated',
+               help=('Check if an Endpoint is activated or requires '
+                     'activation. If it requires activation, exits with '
+                     'status 1, otherwise exits with status 0.'))
+@common_options
+@endpoint_id_arg
+@click.option('--until', type=int,
+              help=('An integer number of seconds in the future. If the '
+                    'endpoint is activated, but will expire by then, exits '
+                    'with status 1'))
+@click.option('--absolute-time', is_flag=True,
+              show_default=True, default=False,
+              help=('Treat the value of --until as a POSIX timestamp, not a '
```

Comment:
Feel free to ignore this, but it might be nice to phrase this as "POSIX timestamp (Unix time)" or "POSIX timestamp (Epoch time)" so that people know what you're talking about and don't get it confused with ISO 8601.